models/document-generation/evaluation-memo.ts

```diff
@@ -0,0 +1,16 @@
+import { EvalPlanMethod, SourceSelection } from "../document-generation";
+
+export interface IEvaluationMemo {
```

Comment:
Can we extend `EvaluationPlan` here instead?

Reply:
Fixed in https://github.com/dod-ccpo/atat-web-api/pull/1710/commits/8daae08831173bdb9f2938699bcc53cf4d572101